include/dfmessages/TimeSync.hpp

```diff
@@ -24,6 +24,9 @@ struct TimeSync
 {
   timestamp_t daq_time{ TypeDefaults::s_invalid_timestamp };        ///< The current DAQ time
   system_time_t system_time{ TypeDefaults::s_invalid_system_time }; ///< The current system time
+  run_number_t run_number{ 0 };
+  size_t sequence_number{ 0 };
```

Comment:
Not a huge issue, but there is potential confusion with dataformats::sequence_number_t, which is only a uint16_t.

Reply:
Good point.  My sense is that it would be better to change the name of this "sequence_number" field instead of changing the type of this sequence_number field to uint16_t.  Maybe "unique_id"?

I also wondered about making the type of the source_pid field pid_t, but I wasn't sure whether that would cause problems with serialization...

Reply:
Thinking about it a bit more, I think I'd vote to leave it as-is, but possibly add `///< Sequence Number of this message, for debugging`

Reply:
Thinking about field sizes and alignment, it may be better to move sequence_number above run_number, change its type to uint64_t, and change source_pid's type to uint32_t.